services/localfile/client/client.go

```diff
@@ -1324,3 +1325,114 @@ func (i *renameCmd) Execute(ctx context.Context, f *flag.FlagSet, args ...interf
 	}
 	return retCode
 }
+
+type mkdirCmd struct {
+	uid      int
+	username string
+	gid      int
+	group    string
+	mode     int
+}
+
+func (*mkdirCmd) Name() string     { return "mkdir" }
+func (*mkdirCmd) Synopsis() string { return "Create a directory" }
+func (*mkdirCmd) Usage() string {
+	return `mkdir --uid=X|username=Y --gid=X|group=Y --mode=X <path>:
+  Create a directory at the specified path.
```

Comment:
Maybe we should mention not supporting automatically creating intermediate directories? (`mkdir -p`)

Reply:
Yes, currently only support creating a directory per action